openIMIS/openIMIS/settings/base.py

```diff
@@ -177,9 +177,23 @@ def SITE_URL():
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
-STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
 STATIC_URL = "/%sstatic/" % SITE_ROOT()
 
+MEDIA_URL = "/file_storage/"
+MEDIA_ROOT = os.path.join(BASE_DIR, "file_storage/")
+
+if not os.path.exists(MEDIA_ROOT):
+    os.makedirs(MEDIA_ROOT)
+
+STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
+    'staticfiles': {
+        'BACKEND': "whitenoise.storage.CompressedManifestStaticFilesStorage",
+    },
+}
+
```

Comment:
@delcroip this came from the following commits:

fe247e8d713382519d88fb9b9079a4dd5f50d503
e77de52280b4d4cbd3f44588cab252cac5d2ff25
21413b441d08f02cd6118b0bdf905e3937ab0c07

Reply:
interesting, I will have a deeper look